Chromatic/ColorModel.swift

```diff
@@ -47,22 +50,19 @@ class ColorModel: NSObject {
             NSCalendarUnit.Minute
             ], fromDate: date)
         
-        self.didUpdate!(timeString: dateString as String, hex: hexString as String, color: color, hours: components.hour, minutes: components.minute)
+        updateBlock(timeString: dateString as String, hex: hexString as String, color: color, hours: components.hour, minutes: components.minute)
```

Comment:
Sorry about the completely unrelated change... but I saw a `!` that was easy to get rid of and I couldn't help myself ;)